eslint.config.js

```diff
@@ -101,16 +95,19 @@ export default tseslint.config(
 			},
 		},
 	},
+	{
+		extends: [tseslint.configs.disableTypeChecked],
+		files: ["**/*.md/*.ts "],
+	},
 	{
 		extends: [vitest.configs.recommended],
 		files: ["**/*.test.*"],
 		rules: {
-			// These on-by-default rules aren't useful in test files.
 			"@typescript-eslint/no-unsafe-assignment": "off",
-			"@typescript-eslint/no-unsafe-call": "off",
```

Comment:
🎉 Very happy that this has since been made unnecessary. I haven't looked deeply, but I'm betting some improvement to Vitest types changed an `any` to something more precise.

For `no-unsafe-assignment`, it's down to two `any` types in Vitest. Filed: https://github.com/vitest-dev/vitest/issues/7015